migrations/20200606105847-convert-collective-long-description-to-html.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+import Promise from 'bluebird';
+import showdown from 'showdown';
+
+const converter = new showdown.Converter();
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const collectives = await queryInterface.sequelize.query(
+      `
+        SELECT * FROM "Collectives"
+        WHERE LENGTH("longDescription") > 0 AND "longDescription" NOT LIKE '<%';
+      `,
+      { type: Sequelize.QueryTypes.SELECT },
+    );
+
+    await Promise.map(collectives, collective =>
+      queryInterface.sequelize.query(
+        `
+        UPDATE "Collectives" c
+        SET "longDescription" = :longDescription
+        WHERE c.id = :id
+      `,
+        {
+          replacements: {
+            longDescription: converter.makeHtml(collective.longDescription),
+            id: collective.id,
+          },
+        },
+      ),
+    );
```

Comment:
As per the comment above, we have ~6000 collectives concerned by this migration. So this code may start a huge number of parallel DB queries. Could we rather have a `for` loop with an `await` inside (or similar) to make sure we do one query at a time?

Also if you have some insights on the time it takes to run locally, that'd be interesting for us to have.